pkg/apiserver/debugapi/param_model_def.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debugapi
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/thoas/go-funk"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/debugapi/endpoint"
+)
+
+var APIParamModelText = &endpoint.BaseAPIParamModel{Type: "text"}
+
+var APIParamModelEscapeText = &endpoint.BaseAPIParamModel{
+	Type: "escape_text",
+	OnResolve: func(v *endpoint.ResolvedValues) error {
+		v.SetValue(url.QueryEscape(v.GetValue()))
+		return nil
+	},
+}
+
+var falselyValues = []string{"false", "0", "null", "undefined", ""}
+
+var APIParamModelBool = &endpoint.BaseAPIParamModel{
+	Type: "bool",
+	OnResolve: func(v *endpoint.ResolvedValues) error {
+		if funk.Contains(falselyValues, v.GetValue()) {
+			v.SetValue("false")
+		} else {
+			v.SetValue("true")
+		}
+		return nil
+	},
+}
+
+var APIParamModelMultiValue = &endpoint.BaseAPIParamModel{
+	Type: "multi_value",
+	OnResolve: func(v *endpoint.ResolvedValues) error {
+		vals := strings.Split(v.GetValue(), ",")
+		v.SetValues(funk.Map(vals, func(str string) string {
+			v, _ := url.QueryUnescape(str)
```

Comment:
Seems that there is an unescape behaviour. What kind of text will user input? Does user need to input an escaped value instead?